config/vufind/shibboleth.ini

```diff
@@ -0,0 +1,11 @@
+; Each section of the file contains IdP configuration.
```

Comment:
Do you think it would make sense to rename this ShibbolethWithWAYF.ini so it matches up with the authentication plugin? That way it is less likely to confuse users of the regular Shibboleth plugin.

Reply:
Yes, it makes more sense, renamed.